software/kottster.yml

```diff
@@ -0,0 +1,15 @@
+name: "Kottster"
+website_url: "https://kottster.app/"
+source_code_url: "https://github.com/kottster/kottster"
+description: "Low-code Node.js admin panel for your app. Connect it to your database, and it will automatically generate pages to view and manage data in your database tables. Get a secure, self-hosted admin panel in minutes."
+licenses:
+  - Apache-2.0
+platforms:
+  - Nodejs
+  - Javascript
+  - Docker
+tags:
+  - Database Management
+  - Content Management Systems (CMS)
+depends_3rdparty: true
```

Comment:
What does this depend on?

Reply:
This flag is more for 3rd party closed source apps such as GitHub, Plex, Mongodb etc. It can be removed here,

Reply:
```suggestion
```